common/bootguard.h

```diff
@@ -180,6 +198,30 @@ typedef struct BG_IBB_ELEMENT_ {
     // BG_IBB_SEGMENT_ELEMENT IbbSegment[];
 } BG_IBB_ELEMENT;
 
+typedef struct BG_IBB_ELEMENT2_ {
+    UINT64                 Tag;
+    UINT8                  Version;
+    UINT8                  : 8;
```

Comment:
This looks ugly to me, why cannot you put Unknown or something alike and avoid the bit count, which is the default for UINT8 anyway?